x/bor/types/util.go

```diff
@@ -93,3 +93,11 @@ func CalcCurrentBorSpanId(latestBorBlock uint64, latestHeimdallSpan *Span) (uint
 
 	return spanId, nil
 }
+
+const (
+	// TODO: Move to params?
```

Comment:
Are we considering this? Is it something we want to make it available for the community to update (e.g. via gov proposal)